contracts/modules/AutoCompoundModule.sol

```diff
@@ -68,8 +71,18 @@ contract AutoCompoundModule is BaseModule {
                 addressHolder.uniswapV3FactoryAddress()
             )
         ).slot0();
+
         //returns true if the value of uncollected fees * 100 is greater than amount in the position * threshold
-        return (((uncollectedFees0 * sqrtPriceX96) / 2**96 + (uncollectedFees1 * 2**96) / sqrtPriceX96) * 100 >
-            ((amount0 * sqrtPriceX96) / 2**96 + (amount1 * 2**96) / sqrtPriceX96) * feesThreshold);
+        // (((tick - tickDelta) / tickSpacing) * tickSpacing, ((tick + tickDelta) / tickSpacing) * tickSpacing);
+        return ((
+            (uncollectedFees0.mul(uint256(sqrtPriceX96)))
+                .div(uint256(2**96).add(uncollectedFees1.mul(uint256(2**96))).div(uint256(sqrtPriceX96)))
```

Comment:
Careful, this is not the same operation as before! Also, the cast uint256(2**96) is really needed?

Reply:
and please remove the comment at line 76